tilt.md

```diff
@@ -0,0 +1,82 @@
+# Rapid iterative development with Tilt
+
+[Tilt](https://tilt.dev) is a tool that enables rapid iterative development of containerized workloads.
+
+Here is an example workflow without Tilt for modifying some source code and testing those changes in a cluster:
+
+1. Modify the source code.
+2. Build the container image.
+3. Either push the image to a registry or load it into your kind cluster.
+4. Deploy all the appropriate Kubernetes manifests for your application.
+    1. Or, if this is an update, you'd instead scale the Deployment to 0 replicas, scale back to 1, and wait for the
+       new pod to be running.
+
+This process can take minutes, depending on how long each step takes.
+
+Here is the same workflow with Tilt:
+
+1. Run `tilt up`
+2. Modify the source code
+3. Wait for Tilt to update the container with your changes
+
+This ends up taking a fraction of the time, sometimes on the order of a few seconds!
+
+## Installing Tilt
+
+Follow Tilt's [instructions](https://docs.tilt.dev/install.html) for installation.
+
+## Installing rukpak and catalogd
+
+operator-controller requires [rukpak](https://github.com/operator-framework/rukpak) and 
+[catalogd](https://github.com/operator-framework/catalogd). Please make sure they're installed, either normally or via 
+their own Tiltfiles, before proceeding. If you want to use Tilt, make sure you specify a unique `--port` flag to each
+`tilt up` invocation.
+
+## Starting Tilt
+
+This is typically a short as:
+
+```shell
+tilt up
```

Comment:
1. Does this need to be run inside the git repo?
2. Does this "take over" the shell/terminal? (i.e. recommend opening another terminal?)

Reply:
Yes and yes